assignment/api/src/java/org/sakaiproject/assignment/api/persistence/AssignmentRepository.java

```diff
@@ -80,4 +81,12 @@ void newSubmission(Assignment assignment,
     long countAssignmentSubmissions(String assignmentId, Boolean graded, Boolean hasSubmissionDate, Boolean userSubmission);
 
     void resetAssignment(Assignment assignment);
+
+    
+    /**
+     * NAM-35
+     */
+    
+	void logMarkerChanges(Assignment assignment, AssignmentMarker oldAssignmentMarker,
```

Comment:
Please change method signature to take an "AssignmentMarker" object as parameter and not a list of parameters.. see  newAssignment(Assignment assignment) as example

Reply:
I used newSubmission as the example here, as we needed the assignment and the AssignmentMarkers (old and new). The context, quotes and modifier we should be able to derive in processing. I can remove them from the parameter list.  Would this be alright?

Reply:
I thought that were ever you call logMarkerChanges from, those values must then be populated on an AssignmentMarkerHistory object, and then the method signature can just look something like this logMarkerChanges(AssignmentMarkerHistory assignmentMarkerHistory ), but I think its fine so will resolve.